docs/API/LorisRESTAPI_v0.0.3.md

```diff
@@ -238,6 +238,30 @@ will return a JSON object of the form:
 
 where 123456, 342332, etc are the candidates that exist for this project.
 
+#### 2.1.5 Single project electrophysiology recordings
+```
+GET /projects/$ProjectName/recordings/
+```
+
+will return a JSON object of the form:
+```js
+{
+  "ElectrophysiologyRecordings": [
```

Comment:
For flexibility, I would change that for `Recordings` and maybe add a `recording_type: Electrophysiology` in the object.
What do you think?